core-web/libs/sdk/experiments/src/lib/dot-experiments.ts

```diff
@@ -63,7 +72,10 @@ export class DotExperiments {
 
             DotExperiments.instance = new DotExperiments(config);
 
+            // Steps
+            this.instance.initializeDatabaseHandler();
             this.instance.setExperimentData();
+            this.instance.initAnalyticsClient();
```

Comment:
Sure that we can't get race conditions here? isn't any of these depends of each other?

Reply:
let me check. But in this PR no.  I changed it a little bit in the next PR.